src/providers/https.ts

```diff
@@ -46,14 +46,37 @@ export function onRequest(
   return _onRequestWithOptions(handler, {});
 }
 
+export interface CallableV1Options {
```

Comment:
We're in the V1 namespace so we don't need to put V1 in the type here. It'll especially draw attention to a V2 that's private when people read the docs.